packages/cli/package.json

```diff
@@ -40,6 +39,7 @@
     "yeoman-test": "^1.7.0"
   },
   "dependencies": {
+    "@loopback/build": "^1.3.2",
```

Comment:
This is not right. We should be using `@loopback/build` installed in the target project to run `lb-prettier`.

Reply:
cli relation command exec `lb-prettier` to fix code style after update of code. If I remove `@loopback/build` from dependencies and lb4 will installed as global and loopback4 application doesn't consists of node_modules, lb4 relation command can't exec lb-prettier.

To resolve this issue `@loopback/build` was added to cli depends.

Reply:
> If I won't add `@loopback/build` in dependencies, will install lb4 globally, and loopback4 application doesn't consists of node_modules, lb4 relation command can't exec lb-prettier to fix formatting of the modified files.

If the target project does not have `lb-prettier` installed (via `@loopback/build`), then we should skip running the command.

> To resolve this issue, `@loopback/build` package was added to cli depends.

Please note that the target project may use different version of `@loopback/build` than the one installed as `@loopback/cli` dependency. As a result, when you invoke `lb-prettier` installed in CLI dependencies, you may reformat the code differently from what is produced by `lb-prettier` used by the target project.

It turns out we already have a solution for this problem, see https://github.com/IsCaster/loopback-next/blob/53bce7f9faf50ccc154348600bc219688f55e93e/packages/cli/lib/base-generator.js#L407-L423

The idea is that users can add `--format` flag if they would like the generator to (re)format the scaffolded code.

```sh
$ lb4 relation --format
```

Please remove the new code you added to invoke `lb-prettier`.

Reply:
Done.  Added --format option into relation cli to run lint:fix.